time/src/main/java/io/spine/time/LocalTimes.java

```diff
@@ -163,4 +142,54 @@ void check(int value) {
             DtPreconditions.checkBounds(value, name().toLowerCase(), 0, upperBound);
```

Comment:
There is a `Parameter` definition above, which uses the constants I've commented on earlier.

We could do as follows:
```
HOURS((int)TimeUnit.DAYS.toHours(1) - 1),
...
```
and avoid using the constants in this file at all.

Reply:
`TimeUnit` returns `long` for all the places where we need `int`. Along with type conversion and warning suppression (`"NumericCastThatLosesPrecision"`), it's too much drama.

I've migrated constant declaration to using `TimeUnit`, but I'd suggest keeping the `Constants` class for the brevity of the code which uses these constants.